src/olympia/devhub/models.py

```diff
@@ -154,6 +154,11 @@ def review_queue(self):
         return (qs.filter(action__in=amo.LOG_REVIEW_QUEUE)
                   .exclude(user__id=settings.TASK_USER_ID))
 
+    def review_log(self):
```

Comment:
After this has landed, I will submit another PR that will clean up this logging mess...this PR focuses on not breaking other things.